src/modules/Profile/components/PersonalAvatar.tsx

```diff
@@ -0,0 +1,99 @@
+import { ChangeEvent } from "react";
+import { Button, Skeleton, notification } from "antd";
+import { useTranslation } from "react-i18next";
+
+import { CameraIcon } from "icons";
+import { useAppDispatch, useAppSelector } from "hooks/useRedux";
+import { RootState } from "redux/store";
+import {
+  resetAfterChangeAvatar,
+  uploadImageActionRequest
+} from "redux/features/profile.slice";
+import { updateProfileActionRequest } from "redux/features/auth.slice";
+import { IPersonalAvatar } from "types/profile.model";
+import { ProfileAvatar, UploadFileAvatar } from "../Profile.styled";
+
+const PersonalAvatar = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const { profile, userInfo } = useAppSelector(
+    (state: RootState) => state.auth
+  );
+  const { uploadedImg, uploadedLoading } = useAppSelector(
+    (state: RootState) => state.profile
+  );
+
+  const handleUploadImage = async (e: ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+
+    if (file) {
+      // check image size > 3MB
+      if (file.size / 1024 / 1024 > 3) {
+        notification.error({
+          message: t("profile:update_avatar"),
+          description: t("profile:upload_size_err")
+        });
+      }
+      // check image format
+      if (!file.type.includes("image")) {
+        notification.error({
```

Comment:
return notification.error

Reply:
why must use return ? notification is function of antd